.github/workflows/build.yml

```diff
@@ -1,5 +1,9 @@
 name: Test
-on: [push, pull_request]
+on:
```

Comment:
nit: if we're touching this file this might be a good PR to also include the exclusions for future changes that will be added in the TODO seciton
```
on:
  push:
    paths:
     - '**'
     - '!**.md'
     - '!LICENSE'
```

Reply:
Good call 👍